src/ssl.c

```diff
@@ -62862,6 +62866,7 @@ int wolfSSL_i2d_PKCS7(PKCS7 *p7, unsigned char **out)
             WOLFSSL_MSG("malloc error");
             goto cleanup;
         }
+        XMEMSET(output, 0, len);
```

Comment:
Why is this needed?

Reply:
Likely not necessary, was just being cautious before passing into wc_PKCS7_EncodeSignedData().  I think we can remove since we're allocating the expected size.